nullaway/src/main/java/com/uber/nullaway/fixserialization/Serializer.java

```diff
@@ -54,6 +64,13 @@ public Serializer(FixSerializationConfig config) {
     initializeOutputFiles(config);
   }
 
+  public static String encodeEscapeSequences(String content) {
```

Comment:
Why not just put this method in `ErrorInfo` (along with the `escapes` map), since that is the only place it is used?  That way both could be private to that class.

Reply:
Also it'd be good to add a bit of Javadoc to this method indicating what is being escaped (maybe with a link to a web page describing the escaping)

Reply:
707889b4cdf57fa473a4a0eb4fa71cbfc5b2d59a